bin/node/runtime/src/weights/pallet_collective.rs

```diff
@@ -13,83 +13,84 @@
 			.saturating_add(DbWeight::get().reads(4 as Weight))
 			.saturating_add(DbWeight::get().writes(4 as Weight))
 	}
 	fn vote(m: u32, ) -> Weight {
-		(46165000 as Weight)
-			.saturating_add((264000 as Weight).saturating_mul(m as Weight))
+		(74460000 as Weight)
+			.saturating_add((290000 as Weight).saturating_mul(m as Weight))
 			.saturating_add(DbWeight::get().reads(2 as Weight))
 			.saturating_add(DbWeight::get().writes(1 as Weight))
 	}
-	// WARNING! Some components were not used: ["b"]
 	fn close_early_disapproved(m: u32, p: u32, ) -> Weight {
-		(52896000 as Weight)
-			.saturating_add((242000 as Weight).saturating_mul(m as Weight))
-			.saturating_add((450000 as Weight).saturating_mul(p as Weight))
+		(86360000 as Weight)
+			.saturating_add((232000 as Weight).saturating_mul(m as Weight))
+			.saturating_add((954000 as Weight).saturating_mul(p as Weight))
 			.saturating_add(DbWeight::get().reads(3 as Weight))
 			.saturating_add(DbWeight::get().writes(3 as Weight))
 	}
 	fn close_early_approved(b: u32, m: u32, p: u32, ) -> Weight {
-		(73235000 as Weight)
-			.saturating_add((247000 as Weight).saturating_mul(m as Weight))
-			.saturating_add((456000 as Weight).saturating_mul(p as Weight))
+		(123653000 as Weight)
 			.saturating_add((1000 as Weight).saturating_mul(b as Weight))
+			.saturating_add((287000 as Weight).saturating_mul(m as Weight))
+			.saturating_add((920000 as Weight).saturating_mul(p as Weight))
 			.saturating_add(DbWeight::get().reads(4 as Weight))
 			.saturating_add(DbWeight::get().writes(3 as Weight))
 	}
-	// WARNING! Some components were not used: ["b"]
 	fn close_disapproved(m: u32, p: u32, ) -> Weight {
-		(56171000 as Weight)
-			.saturating_add((250000 as Weight).saturating_mul(m as Weight))
-			.saturating_add((462000 as Weight).saturating_mul(p as Weight))
+		(95395000 as Weight)
+			.saturating_add((236000 as Weight).saturating_mul(m as Weight))
+			.saturating_add((965000 as Weight).saturating_mul(p as Weight))
 			.saturating_add(DbWeight::get().reads(4 as Weight))
 			.saturating_add(DbWeight::get().writes(3 as Weight))
 	}
 	fn close_approved(b: u32, m: u32, p: u32, ) -> Weight {
-		(77594000 as Weight)
-			.saturating_add((246000 as Weight).saturating_mul(m as Weight))
-			.saturating_add((453000 as Weight).saturating_mul(p as Weight))
-			.saturating_add((2000 as Weight).saturating_mul(b as Weight))
+		(135284000 as Weight)
+			.saturating_add((4000 as Weight).saturating_mul(b as Weight))
+			.saturating_add((218000 as Weight).saturating_mul(m as Weight))
+			.saturating_add((951000 as Weight).saturating_mul(p as Weight))
 			.saturating_add(DbWeight::get().reads(5 as Weight))
 			.saturating_add(DbWeight::get().writes(3 as Weight))
 	}
 	fn disapprove_proposal(p: u32, ) -> Weight {
-		(0 as Weight)
-			.saturating_add((490000 as Weight).saturating_mul(p as Weight))
+		(50500000 as Weight)
+			.saturating_add((966000 as Weight).saturating_mul(p as Weight))
```

Comment:
that's a lot more weight than I expected, what do you think where it comes from?